### ISSUE TITLE
ManagenentRepair nemesis: 'ascii' codec can't decode byte 0xe2 in position 0: ordinal not in range(128)

### ISSUE THREAD

**Issue by juliayakovlev** [scylladb/scylla-cluster-tests#919]
ManagenentRepair nemesis failure during MV backpressure longevity test ([http://jenkins.cloudius-systems.com:8080/job/scylla-staging/job/scylla-staging-longevity-2mv-backpressure-4d-test/label=aws-scylla-qa-builder3/6/console]) 

```
2019-04-21 08:51:33,170 sct_events       L0336 INFO | 2019-04-21 08:51:33.159: (DisruptionEvent Severity.ERROR): name=ManagementRepair node=Node longevity-2mv-backpressure-4d-VERSION-db-node-28db9f55-5 [54.172.11.152 | 172.30.0.138] (seed: False) duration=30 error='ascii' codec can't decode byte 0xe2 in position 0: ordinal not in range(128)
Traceback (most recent call last):
  File "/sct/sdcm/nemesis.py", line 887, in wrapper
    result = method(*args, **kwargs)
  File "/sct/sdcm/nemesis.py", line 1063, in disrupt
    self.call_random_disrupt_method()
  File "/sct/sdcm/nemesis.py", line 509, in call_random_disrupt_method
    disrupt_method()
  File "/sct/sdcm/nemesis.py", line 744, in disrupt_mgmt_repair_cli
    manager_tool = mgmt.get_scylla_manager_tool(manager_node=manager_node)
  File "/sct/sdcm/mgmt.py", line 460, in get_scylla_manager_tool
    return ScyllaManagerToolRedhatLike(manager_node=manager_node)
  File "/sct/sdcm/mgmt.py", line 618, in __init__
    ScyllaManagerTool.__init__(self, manager_node=manager_node)
  File "/sct/sdcm/mgmt.py", line 476, in __init__
    logger.debug("Initiating Scylla-Manager, version: {}".format(self.version))
  File "/sct/sdcm/mgmt.py", line 486, in version
    return self.sctool.run(cmd=cmd, is_verify_errorless_result=True)
  File "/sct/sdcm/mgmt.py", line 708, in run
    res = self.parse_result_table(res=res)
  File "/sct/sdcm/mgmt.py", line 718, in parse_result_table
    not (line.startswith('╭') or line.startswith('├') or line.startswith(
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 0: ordinal not in range(128)
```

http://jenkins.cloudius-systems.com:8080/job/enterprise-2019.1/job/longevity-tests/job/longevity-mv-si-4d-test/label=aws-eu-west1-qa-builder3/11/console

**Comment by roydahan** [comment]
I think we know about this, the ami for monitor isn't correct and manager isn't installed.

**Comment by juliayakovlev** [comment]
But it's not inform about this, it raise the failure. I think we need to handle this situation and don't run this nemesis, it manager isn't installed

**Comment by fruch** [comment]
I've seen this too, while port the test case to the new configuration:
```
Reproduced traceback from: /home/fruch/Projects/scylla-cluster-tests/.venv/local/lib/python2.7/site-packages/avocado/core/test.py:449
Traceback (most recent call last):
  File "/home/fruch/Projects/scylla-cluster-tests/mgmt_cli_test.py", line 99, in test_manager_sanity
    self.test_mgmt_cluster_crud()
  File "/home/fruch/Projects/scylla-cluster-tests/mgmt_cli_test.py", line 57, in test_mgmt_cluster_crud
    manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
  File "/home/fruch/Projects/scylla-cluster-tests/sdcm/mgmt.py", line 461, in get_scylla_manager_tool
    return ScyllaManagerToolRedhatLike(manager_node=manager_node)
  File "/home/fruch/Projects/scylla-cluster-tests/sdcm/mgmt.py", line 619, in __init__
    ScyllaManagerTool.__init__(self, manager_node=manager_node)
  File "/home/fruch/Projects/scylla-cluster-tests/sdcm/mgmt.py", line 477, in __init__
    logger.debug("Initiating Scylla-Manager, version: {}".format(self.version))
  File "/home/fruch/Projects/scylla-cluster-tests/sdcm/mgmt.py", line 487, in version
    return self.sctool.run(cmd=cmd, is_verify_errorless_result=True)
  File "/home/fruch/Projects/scylla-cluster-tests/sdcm/mgmt.py", line 709, in run
    res = self.parse_result_table(res=res)
  File "/home/fruch/Projects/scylla-cluster-tests/sdcm/mgmt.py", line 719, in parse_result_table
    not (line.startswith('╭') or line.startswith('├') or line.startswith(
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 0: ordinal not in range(128)
```

**Comment by aleksbykov** [comment]
This should be fixed by #973
